docs/en/changes/changes.md

```diff
@@ -78,6 +78,7 @@
 * Fix TCP service instances are lack of instance properties like `pod` and `namespace`, which causes Pod log not to work for TCP workloads.
 * Add Python HBase happybase module component ID(94).
 * Fix gRPC alarm cannot update settings from dynamic configuration source.
+* [**Breaking Change**] Optimize single trace query performance by customizing routing in ElasticSearch
```

Comment:
```suggestion
* [**Breaking Change**] Optimize single trace query performance by customizing routing in ElasticSearch. SkyWalking trace segments and Zipkin spans are using trace ID for routing.
```

Reply:
So, does this breaking mean, the existing trace segments/spans are not searchable anymore? Due to the routing is changed, it most likely can't hit the records.

Reply:
yes